Connectors/Rhino/Speckle.Connectors.RhinoShared/Operations/Receive/RhinoHostObjectBuilder.cs

```diff
@@ -208,8 +208,6 @@ CancellationToken cancellationToken
             conversionResults.Add(new(Status.SUCCESS, obj, r, result.GetType().ToString()));
             bakedObjectIds.Add(r);
           }
-
-          applicationIdMap[objectId] = conversionIds;
```

Comment:
@clairekuang this was bad juju change - wondering why it happened? I'll fix it on my end now (in this branch)